packages/shared/src/components/Feed.tsx

```diff
@@ -208,13 +207,24 @@ export default function Feed<T>({
     };
   }, [postMenuLocation, virtualizedNumCards, postModalIndex]);
 
+  const onRemovePost = useCallback(
+    async (removePostIndex: number) => {
+      const item = items[removePostIndex] as PostItem;
+      removePost(item.page, item.index);
+    },
+    [items, removePost],
+  );
+
   const feedContextValue = useMemo(() => {
```

Comment:
Moved stuff to the shared active feed context so we don't need the million layers of prop drilling.
(And those that could just be from bottom layer didn't even need the prop drilling)

Reply:
Yeah, we should actually start doing this whenever we get the chance.